XRTK-Core/Packages/com.xrtk.core/Runtime/Interfaces/Providers/Controllers/Hands/IMixedRealityHandControllerDataProvider.cs

```diff
@@ -1,30 +1,36 @@
 ﻿// Copyright (c) XRTK. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System.Collections.Generic;
 using XRTK.Definitions.Controllers.Hands;
 
 namespace XRTK.Interfaces.Providers.Controllers.Hands
 {
     public interface IMixedRealityHandControllerDataProvider : IMixedRealityControllerDataProvider
     {
         /// <summary>
-        /// Should hand mesh data be read if the platform provides it?
+        /// Gets the current rendering mode for hand controllers.
         /// </summary>
-        bool HandMeshingEnabled { get; }
+        HandRenderingMode RenderingMode { get; set; }
 
         /// <summary>
         /// Are hand physics enabled?
         /// </summary>
-        bool HandPhysicsEnabled { get; }
+        bool HandPhysicsEnabled { get; set; }
 
         /// <summary>
         /// Shall hand colliders be triggers?
         /// </summary>
-        bool UseTriggers { get; }
+        bool UseTriggers { get; set; }
 
         /// <summary>
         /// Gets the configured hand bounds mode to be used with hand physics.
         /// </summary>
-        HandBoundsMode BoundsMode { get; }
+        HandBoundsMode BoundsMode { get; set; }
+
+        /// <summary>
+        /// Gets the list of poses tracked by the data provider.
+        /// </summary>
+        IReadOnlyList<HandControllerPoseProfile> TrackedPoses { get; }
```

Comment:
I'm still not sure how I feel about the profiles being able to be accessed. I think ideally we'd like to make each pose more akin to an InputAction than a profile.

Reply:
Thanks, I missed this. I'll make sure to remove it.